test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -224,13 +234,299 @@ trait TestData extends ModelGenerators {
+          outstandingAmount = Some(BigDecimal("9000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("9000")
+            )
+          )
+        )
+      )
+    )
+
+  val singlePartiallyOutstandingReturn: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReferenceGen.sample.get),
+          originalAmount = BigDecimal("9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("5000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("5000")
+            ),
+            FinancialTransactionItem(
+              subItem = "001",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("4000")
+            )
+          )
+        )
+      )
+    )
+
+  val twoLineItemPartiallyOutstandingReturn: FinancialTransactionDocument = {
+    val sapDocumentNumber = sapDocumentNumberGen.sample.get
+    val chargeReference   = chargeReferenceGen.sample.get
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumber,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReference),
+          originalAmount = BigDecimal("9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("5000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("5000")
+            ),
+            FinancialTransactionItem(
+              subItem = "001",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("4000")
+            )
+          )
+        ),
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumber,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReference),
+          originalAmount = BigDecimal("2000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("2000")),
```

Comment:
This FinancialTransaction is fully paid so won't be returned from the api

Reply:
It's not, it's outstanding 2000 - the name of the example needs changing though.